Revit_Core_Engine/Convert/Revit/Internal/ToSolid.cs

```diff
@@ -95,6 +95,40 @@ public static Solid ToSolid(this Element scopeBox, RevitSettings settings = null
         }
 
         /***************************************************/
+
+        [Description("Converts Revit bounding box to a Revit Solid.")]
+        [Input("bbox", "Revit bounding box to be converted.")]
+        [Input("settings", "Revit adapter settings to be used while performing the convert.")]
+        [Output("solid", "Revit Solid resulting from converting the input Revit scope box.")]
+        public static Solid ToSolid(this BoundingBoxXYZ bbox, RevitSettings settings = null)
+        {
+            XYZ pt0 = new XYZ(bbox.Min.X, bbox.Min.Y, bbox.Min.Z);
+            XYZ pt1 = new XYZ(bbox.Max.X, bbox.Min.Y, bbox.Min.Z);
+            XYZ pt2 = new XYZ(bbox.Max.X, bbox.Max.Y, bbox.Min.Z);
+            XYZ pt3 = new XYZ(bbox.Min.X, bbox.Max.Y, bbox.Min.Z);
+
+            Line edge0 = Line.CreateBound(pt0, pt1);
+            Line edge1 = Line.CreateBound(pt1, pt2);
+            Line edge2 = Line.CreateBound(pt2, pt3);
+            Line edge3 = Line.CreateBound(pt3, pt0);
+
+            List<Curve> edges = new List<Curve>();
+            edges.Add(edge0);
+            edges.Add(edge1);
+            edges.Add(edge2);
+            edges.Add(edge3);
+
+            double height = bbox.Max.Z - bbox.Min.Z;
+
+            CurveLoop baseLoop = CurveLoop.Create(edges);
+
+            List<CurveLoop> loopList = new List<CurveLoop>();
+            loopList.Add(baseLoop);
+
+            Solid solid = GeometryCreationUtilities.CreateExtrusionGeometry(loopList, XYZ.BasisZ, height);
+            return SolidUtils.CreateTransformed(solid, bbox.Transform);
```

Comment:
Sanity check - I just looked up the API description and it sounds a bit like you would need to take inverse transform here... Have you tested this bit?
![image](https://user-images.githubusercontent.com/26874773/179224302-8c91b32f-8812-4386-9ec0-bfb45a602e2f.png)

Reply:
After a discussion with @michal-pekacki we came to a conclusion that the code is correct, while the API docs are a bit ambiguous.

Reply:
I tested that and it works correctly.